nvtabular/workflow.py

```diff
@@ -158,7 +158,7 @@ def fit(self, dataset: Dataset):
 
     def fit_transform(self, dataset: Dataset) -> Dataset:
         """Convenience method to both fit the workflow and transform the dataset in a single
-        call. Equivalent to calling workflow.fit(dataset) followed by workflow.transform(dataset)
+        call. Equivalent to calling ``workflow.fit(dataset)`` followed by ``workflow.transform(dataset)``
```

Comment:
This line is too long as is tripping up the format checker:

```suggestion
        call. Equivalent to calling ``workflow.fit(dataset)`` followed by
        ``workflow.transform(dataset)``
```